### DISCUSSION TITLE
希望下载的视频可以保留原标题

### DISCUSSION THREAD

**Discussion by WhiteSheep-y** [the1812/Bilibili-Evolved#3062] [功能建议 / Ideas]
下载好后的视频文件标题是数字，而非原视频标题，如果不重命名文件将给辨认视频造成很大困难，极不友好
<img width="287" alt="图片" src="https://user-images.githubusercontent.com/72679569/155832125-fa585bc8-569c-4196-a002-c00319d929e5.png">

**Comment by the1812** [comment]
选择其他输出方式都是有命名的
